modules/analysis/src/endpoints/test.rs

```diff
@@ -613,3 +613,45 @@ async fn issue_tc_2054(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
 
     Ok(())
 }
+
+#[test_context(TrustifyContext)]
+#[test(actix_web::test)]
+async fn issue_spdx_package_of(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
+    // test case for the simple case of "relationshipType": "PACKAGE_OF" spdx relationships:
+    // https://github.com/trustification/trustify/issues/1140
+
+    let app = caller(ctx).await?;
+    ctx.ingest_document("spdx/SATELLITE-6.15-RHEL-8.json")
+        .await?;
+
+    let purl = "pkg:rpm/redhat/rubygem-google-cloud-compute@0.5.0-1.el8sat?arch=src";
+
+    // Ensure parent has deps that include the child
+    let uri = format!("/api/v2/analysis/dep/{}", urlencoding::encode(purl));
+    let request: Request = TestRequest::get().uri(&uri).to_request();
+    let response: Value = app.call_and_read_body_json(request).await;
+    log::debug!("{response:#?}");
+
+    let sbom = &response["items"][0];
+    let matches: Vec<_> = sbom["deps"]
+        .as_array()
+        .into_iter()
+        .flatten()
+        .filter(|m| {
+            m == &&json!({
+                "sbom_id": sbom["sbom_id"],
+                "node_id": "SPDXRef-83c9faa0-ca85-4e48-9165-707b2f9a324b",
+                "relationship": "PackageOf",
+                "purl": [],
+                "cpe": m["cpe"], // long list assume it's correct
```

Comment:
I'm good if you are, but worth filtering on something to target a specific element to make sure it's there?

Reply:
Isn't this doing filtering?

Reply:
I was referring to the comment. Is there anything about the "long list" that you might possibly filter to find something in it that verifies the test? Or is enough just to assume it's correct if it's "long"? It may be overkill, but the comment gives me pause.

Reply:
oh I see, yeah I think including that would just add a bunch of noise to the test.  I think matching on the other object properties is plenty good enough.